webui/ec2/EC2.py

```diff
@@ -78,15 +78,13 @@ def delete_key(key_id):
     if user is None:
         return False
     if len(user.keys) == 1:
-        delete_user(user.name)
-        return True
+        return delete_user(user.name)
     else:
         iam_res = boto3.resource(
             'iam', aws_access_key_id=provider_conf.EC2['key'],
             aws_secret_access_key=provider_conf.EC2['secret'])
         key = iam_res.AccessKey(user.name, key_id)
-        key.delete()
-        return True
+        return key.delete()
```

Comment:
If I'm not wrong `delete()` doesn't return something
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.AccessKey.delete